docs/user-guide/project.md

```diff
@@ -156,6 +156,36 @@ If you want to define a property which can have more than one value, you should
+
+### Widening or restricting a cardinality of a property without data
+
+If there is no data yet available, widening or restricting the cardinality can be relatively straightforward.
+Widening the cardinality of a data model refers to changing the relationship between resource classes in a way that allows more instances of a resource class to be associated with another resource class. Restricting the cardinality of a data model refers to limiting the number of instances of a resource class that can be associated with another resource class. Here are the steps to changing the cardinality of a data model when there is no data yet available:
+
+In this example activate the toggle "multiple" for widening the cardinality:
+![Changing a cardinality of a property without data](../assets/images/changing_without_data_1.png)
+
+The pop-up window informs you that you are going to change the cardinality and you may not be able to reverse it:
+![Changing a cardinality of a property without data](../assets/images/changing_without_data_2.png)
+
+If you want to confirm the change, click "Continue" and the toggle will now be activated. The same procedure applies to the toggle "required":
+![Changing a cardinality of a property without data](../assets/images/changing_without_data_3.png)
+
+### Widening or restricting a cardinality of a property with data
+If data is already present, widening and restricting the cardinality of a property is possible if e.g., in a required property in every resource the values are available. In this case the same procedure applies as described in the sections above.
+
+### Widening or restricting a cardinality of a property if a resource does not contain all the values
+If values are missing in the resources and you want to adjust the cardinality so that the resources do not match the selected cardinality (e.g. set a property to required, but the field is empty), an error message is displayed. In this case, all resources must be corrected first, then the desired cardinality can be set.
+
+### Widening or restricting a cardinality of a property: Special cases
+#### Superclasses and subclasses
+If the cardinality of a superclass is changed, then the subclass is not affected and vice versa the same applies. In general, the cardinality of a subclass can be more or equally restrictive than the cardinality of a superclass.
+
+#### Inherited properties with different naming
+Changing the cardinalities of a superclass and subclass and reversing them does not work if the inherited properties are named differently.
```

Comment:
Result of the discussion with Christian: Changing the cardinality of a superclass and a subclass and reversing it does not work if the properties are in a subproperty relationship or the changed property in the superclass does not exist in the subclass.

Reply:
thx! Done.